composer.json

```diff
@@ -186,6 +186,10 @@
         "Errors when using with modules that alter a section's contextual menu": "https://www.drupal.org/files/issues/2020-05-22/3106939-4.patch",
         "Indicate Drupal 8/9 compatibility via core_version_requirement": "https://www.drupal.org/files/issues/2020-05-22/d9_compatibility-3138698-4.patch"
       },
+      "drupal/upgrade_status": {
+        "Exclude node_modules from scan": "https://www.drupal.org/files/issues/2020-08-03/3162997-6.patch"
```

Comment:
The patch still needs work to be accepted, better not spread incomplete (WIP) ones

But great to share as WIP pr! Thank you!

Reply:
Removing changes to composer file as no relation to this PR